src/components/MessageInput.jsx

```diff
@@ -0,0 +1,107 @@
+/** @jsxImportSource @emotion/react */
+import { useState, useRef } from 'react';
+import { css } from '@emotion/react';
+import Search from '/public/assets/search.png';
+import Send from '/public/assets/send.png';
```

Comment:
`public` 폴더에서 import 해 오는 것들은 경로에 `/public`을 명시하지 않아도 됩니다 ㅎㅎ
```suggestion
import Search from '/assets/search.png';
import Send from '/assets/send.png';
```